### PR TITLE
fix: disable sourcemaps when building storybook

### PR DESCRIPTION
**Pull request by montezume**
To try to help with netlify build issues.

### CODE REVIEW

**Comment by tdeekens** [comment]
Update: sourcemaps add 30 seconds. Yay, guess was correct. 63 vs 35 secs.

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -95,5 +106,7 @@ module.exports = ({ config }) => {
     },
   ];
 
+  config.optimization.minimizer[0].options.sourceMap = disableSourceMaps;
```
Comment (montezume) [inline .storybook/webpack.config.js]: hm,  but we just set minimizer to empty array. How can we access this property now?

Comment (tdeekens) [reply .storybook/webpack.config.js]: No. We keep the minimizer in my commit. We just disable source maps emission.

Comment (tdeekens) [reply .storybook/webpack.config.js]: I intended to actually use terser but it to not emit any source maps. Fixed up my commit.

**Comment by montezume** [comment]
Starting to think that any improvements we make is just rearranging deck chairs on the 🚢 . The plan is the problem, not our build 🤔

**Comment by tdeekens** [comment]
Yeah feels like an angle not working or yielding enough. So either up the plan to see if that helps or maybe truely build on circleci. will have to think about it. maybe it can even help having a version switcher. it is a bigger leap obviously though.

**Pull request closed without merging**